config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
-  resources :books
-  resources :tags
-  root 'books#index'
+    resources :books do
+        # put :reserve, on: :member
```

Comment:
do we need these?

Reply:
Nope, I've refactored the code so that a reservation is created/destroyed when a book gets updated -- I've now removed the unused resources.